pycloudlib/ibm/cloud.py

```diff
@@ -403,6 +403,17 @@ def _get_or_create_key(self) -> str:
         self.created_keys.append(key_id)
         return key_id
 
+    def get_image_id_by_name(self, name: str) -> str:
```

Comment:
Nice! What sorts of images are you testing? Think that this is something we might want for other clouds? I can imagine this being useful, for example, to test cloud-init for non-ubuntu clouds.

Maybe we should put an abstract method in the base class as well so we can have a standardized interface for other clouds to match?

Reply:
let me take a look at that! would definitely be something that would be both easy to add to all clouds and also just generally useful.